app/client/components/creditTransfer/creditTransferList.jsx

```diff
@@ -80,17 +80,17 @@ class CreditTransferList extends React.Component {
     this.setState({isLoading: false})
   }
 
-  componentDidUpdate(newProps) {
+  componentDidUpdate(prevProps) {
     let { credit_transfer_ids, item_list } = this.props;
 
     // handles credit_transfer_ids array
-    if (credit_transfer_ids !== newProps.credit_transfer_ids) {
+    if (credit_transfer_ids !== prevProps.credit_transfer_ids) {
       this.setState({credit_transfer_ids: {}});
-      // this.sortCreditTransfers(newProps.credit_transfer_ids)
+      this.sortCreditTransfers(credit_transfer_ids)
     }
 
```

Comment:
do we need to optimize credit transfer list like we did on the transfer account @enjeyw ?

Reply:
we should, but given it only shows the last 250 it's not such an issue